### PR TITLE
update documentation MenuBlock config

### PR DESCRIPTION
**Pull request by hug963**
## Document MenuBlock use 

The configuration for the MenuBlock was changed here https://github.com/sonata-project/SonataBlockBundle/commit/347668a81e0f81246c897eefe72689196758019e and never documented.

This is the only way the MenuBlock can be used.

### CODE REVIEW

File: Resources/doc/reference/provided_blocks.rst [hunk 1]
```diff
@@ -33,10 +33,21 @@ MenuBlockService
...
+                'footer': 'Footer' # Use alias name if using menus as a service
+
+Upon configuration, you may set some rendering options (see KNP Doc for those).
 
 Set ``cache_policy`` to private if this menu is dedicated to be in a user part.
 
 A specific menu template is provided as well to render Bootstrap3's side menu, you may use it by setting the ``menu_template`` option to ``SonataBlockBundle:Block:block_side_menu_template.html.twig`` (see the implementation in SonataUserBundle or Sonata's e-commerce suite).
 
-.. _KnpMenuBundle documentation: https://github.com/KnpLabs/KnpMenuBundle/blob/master/Resources/doc/index.md#rendering-menus
+.. _KnpMenuBundle documentation: http://symfony.com/doc/current/bundles/KnpMenuBundle/index.html
```
Comment (core23) [inline Resources/doc/reference/provided_blocks.rst]: symfony supports https too

Comment (hug963) [inline Resources/doc/reference/provided_blocks.rst]: done!

**Comment by core23** [comment]
LGTM

**Comment by OskarStark** [comment]
@sonata-project/contributors any idea why travis is failing?

**Comment by greg0ire** [comment]
No. I just re tried this particular job, for science.

**Comment by greg0ire** [comment]
Aaaaand it's green.

File: Resources/doc/reference/provided_blocks.rst [hunk 2]
```diff
@@ -33,10 +33,21 @@ MenuBlockService
...
+Provide a list of available menus in the configuration.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        sonata_block:
+            menus:
+                'AppBundle:Builder:mainMenu': 'Main Menu'
+                'footer': 'Footer Menu' # Use alias name if using menus as a service
```
Comment (greg0ire) [inline Resources/doc/reference/provided_blocks.rst]: "menu as a service", or "menus as services". Please pick one.

**Comment by OskarStark** [comment]
Thank you @hug963 👍

File: Resources/doc/reference/provided_blocks.rst (hunk 2 above)
Comment (greg0ire) [inline Resources/doc/reference/provided_blocks.rst]: Well… next time I guess

Comment (OskarStark) [inline Resources/doc/reference/provided_blocks.rst]: oh sorry @greg0ire

Comment (greg0ire) [inline Resources/doc/reference/provided_blocks.rst]: No big deal :wink:

**Pull request merged**